docs/catalog/cloud/Design for independent scaling of logical components.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Design for independent scaling of logical components
+
+## Description
+A microservice architecture may reduce the compute resources required, as it allows for independent scaling of its logical components and ensures they are scaled according to the demand.
+
+## Solution
+
+Use [Dapr](https://dapr.io) Framework or other cloud native projects to help you separate your application functionality into different microservices, to allow independent scaling of its logical components.
```

Comment:
nit: Since patterns are supposed to be vendor-agnostic, consider changing this to something like "Use cloud native projects like Dapr to help you separate...". This phrasing provides the vendor-specific solution as an option to the generic approach instead of implying that Dapr is THE solution here.